Cargo.toml

```diff
@@ -14,6 +14,7 @@ readme = "README.md"
 repository = "https://github.com/tauri-apps/tao"
 documentation = "https://docs.rs/tao"
 categories = [ "gui" ]
+exclude = ["audits/**"]
```

Comment:
we'll still end up with a lot of unnecessary stuff (though small in size), what about
```suggestion
include = ["/README.md", "src/**/*.rs", "LICENSE*"]
```
(cargo.toml/.lock are included by cargo anyway)

Reply:
ohh good question 🤔